<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SelectReceiver</class>
 <widget class="QWidget" name="SelectReceiver">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>700</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>700</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>选择联系人</string>
  </property>
  <property name="styleSheet">
   <string notr="true">/* 主窗口样式 */
QWidget {
    background-color: #f0f5ff;
    font-family: 'Microsoft YaHei';
}

/* 容器样式 */
#mainContainer {
    background-color: white;
    border-radius: 15px;
    padding: 25px;
    margin: 25px;
    box-shadow: 0 6px 12px rgba(0,0,0,0.12);
}

/* 标签样式 */
QLabel {
    font-size: 16px;
    color: #444444;
    font-weight: bold;
}

/* 文本编辑框样式 */
QTextEdit {
    background-color: white;
    border: 2px solid #c5d9f1;
    border-radius: 10px;
    padding: 15px;
    font-size: 14px;
    color: #333333;
}

/* 输入框样式 */
QLineEdit {
    border: 2px solid #c5d9f1;
    border-radius: 10px;
    padding: 15px;
    font-size: 14px;
    color: #222222;
    background-color: white;
}

QLineEdit:focus {
    border: 2px solid #4a90e2;
}

/* 下拉框样式 */
QComboBox {
    border: 2px solid #c5d9f1;
    border-radius: 10px;
    padding: 15px;
    font-size: 14px;
    color: #222222;
    background-color: white;
}

QComboBox:focus {
    border: 2px solid #4a90e2;
}

QComboBox::drop-down {
    subcontrol-origin: padding;
    subcontrol-position: top right;
    width: 30px;
    border-left-width: 1px;
    border-left-color: #c5d9f1;
    border-left-style: solid;
    border-top-right-radius: 10px;
    border-bottom-right-radius: 10px;
}

/* 按钮通用样式 */
QPushButton {
    border: none;
    border-radius: 10px;
    padding: 12px 20px;
    font-size: 16px;
    font-weight: bold;
    min-width: 100px;
}

/* 确定按钮样式 */
#confirmButton {
    background-color: #4a90e2;
    color: white;
}

#confirmButton:hover {
    background-color: #3a7bc8;
}

#confirmButton:pressed {
    background-color: #2c6bb6;
}

/* 取消按钮样式 */
#quitButton {
    background-color: #8a8a8a;
    color: white;
}

#quitButton:hover {
    background-color: #7a7a8a;
}</string>
  </property>
  <widget class="QWidget" name="mainContainer" native="true">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>30</y>
     <width>600</width>
     <height>540</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="Label1">
      <property name="text">
       <string>以下是您可以联系的用户：</string>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="verticalSpacer_1">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>10</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QTextEdit" name="receiverTextEdit">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>1</verstretch>
       </sizepolicy>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="verticalSpacer_2">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QLabel" name="Label1_2">
      <property name="text">
       <string>您也可以在这里选择：</string>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="verticalSpacer_3">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>10</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QComboBox" name="receiverComboBox">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="verticalSpacer_4">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QLabel" name="Label2">
      <property name="text">
       <string>请输入您想要联系的对象：</string>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="verticalSpacer_5">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>10</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QLineEdit" name="msgLineEdit">
      <property name="placeholderText">
       <string>请输入您想要联系的对象</string>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="verticalSpacer_6">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>30</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <layout class="QHBoxLayout" name="buttonLayout">
      <item>
       <widget class="QPushButton" name="confirmButton">
        <property name="text">
         <string>确定</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="quitButton">
        <property name="text">
         <string>取消</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
